frontend/src/components/SiteNav.js

```diff
@@ -62,18 +61,20 @@ const SiteNav = ({
   }, [hasAlerts]);
 
   // Determine Default Region.
-  const regions = allRegionsUserHasPermissionTo(user).join();
+  const regions = allRegionsUserHasPermissionTo(user).join(', ');
   const defaultRegion = user.homeRegionId || regions[0] || 0;
   const hasMultipleRegions = regions && regions.length > 1;
 
   // If user has more than one region, Regions label is plural, else singular
   const regionLabel = () => {
+    if (defaultRegion === 14) {
+      return 'All Regions';
+    }
+
     if (hasMultipleRegions) {
       return `Regions ${regions}`;
     }
-    if (defaultRegion === 14) {
-      return 'All Regions;';
-    }
```

Comment:
Flipped the order here since putting "hasMultipleRegions" first means you'd basically never hit that second clause. My understanding is that the "defaultRegion if" should take precendence.

Also removed a semicolon typo